export const handleError = err => {
    console.warn('Error in Promise: ', err);
};
